.github/ISSUE_TEMPLATE/announcement.md

```diff
@@ -26,7 +26,7 @@ assignees: ''
 - [ ] macOS 10.13
 - [ ] macOS 10.14
 - [ ] macOS 10.15
-- [ ] macOS 11.0
+- [ ] macOS 11
```

Comment:
Do we need to rename it in the [bug-report.md](https://github.com/actions/virtual-environments/blob/main/.github/ISSUE_TEMPLATE/bug-report.md) and [tool-request.md](https://github.com/actions/virtual-environments/blob/main/.github/ISSUE_TEMPLATE/tool-request.md) as well?

Reply:
Done